.github/ISSUE_TEMPLATE/promotion.md

```diff
@@ -7,6 +7,7 @@ _start: 2024-02-27T09:00
 _period: 7 days
 ---
 - [ ] The title of this issue matches `Promotion yyyy-mm-dd`
+- [ ] The sprint of this issue matches the date in the title
```

Comment:
Consider …
```suggestion
- [ ] The sprint of this issue corresponds to the date in the title
```
… and moving it to the bottom. We may not have a sprint until after the promotion is decided.